sql/src/test/java/org/apache/druid/sql/calcite/BaseCalciteQueryTest.java

```diff
@@ -295,15 +298,11 @@ public static Map<String, Object> getTimeseriesContextWithFloorTime(
   public final SqlEngine engine0;
   final boolean useDefault = NullHandling.replaceWithDefault();
 
-  @Rule(order = 1)
-  public ExpectedException expectedException = ExpectedException.none();
-
   @Rule(order = 2)
```

Comment:
We can also remove the junit rule order or re-order the rules (same applies to `@Rule(order = 3) queryFrameworkRule`:
```suggestion
  @Rule
```